libs/repositories/src/datasources/coingecko.ts

```diff
@@ -6,14 +6,217 @@ import type { components, paths } from '../gen/coingecko/coingecko-pro-types';
 
 export const COINGECKO_PRO_BASE_URL = 'https://pro-api.coingecko.com';
 
-export const COINGECKO_PLATFORMS: Record<SupportedChainId, string | undefined> =
-  {
-    [SupportedChainId.MAINNET]: 'ethereum',
-    [SupportedChainId.GNOSIS_CHAIN]: 'xdai',
-    [SupportedChainId.ARBITRUM_ONE]: 'arbitrum-one',
-    [SupportedChainId.BASE]: 'base',
-    [SupportedChainId.SEPOLIA]: undefined,
-  };
+/**
+ * Map of chain IDs to Coingecko platform IDs, for every platform that has a network id.
+ * The platform ID is used to identify the blockchain on Coingecko.
+ */
+export const COINGECKO_PLATFORMS: Record<number, string | undefined> = {
```

Comment:
Map chain ids to coingecko ids, if any.

Reply:
How did you generate this file? 

Maybe link that in the comments. Its likely this needs maintenance

Reply:
Fixed.
Also added the data source

Reply:
Good point. I did it in the console as a one off.
I can add a script to update it.